test/e2e/helpers/setupConnection.ts

```diff
@@ -1,32 +1,58 @@
+import { expect } from '@playwright/test'
 import 'reflect-metadata'
-import { fetchPost } from '../../helpers/routeHelper.js'
+
+import { fetchGet, fetchPost } from '../../helpers/routeHelper.js'
 import { checkEmails, extractInvite, extractPin, getHostPort } from './smtpEmails.js'
 
-export async function withConnection(smtp4devUrl: string, aliceUrl: string, bobUrl: string) {
-  // Send an invite from Alice to Bob
-  await fetchPost(`${aliceUrl}/connection/new/create-invitation`, {
+const uuidRegex = /[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-4[0-9a-fA-F]{3}-[89ab][0-9a-fA-F]{3}-[0-9a-fA-F]{12}/g
+
+export async function withConnection(issuerUrl, holderUrl) {
```

Comment:
why are you reducing the configurability of this? Also you don't seem to have updated it's usage

also I understand the desire to be generic beyond `alice` and `bob` but `holder` and `issuer` aren't the right terms either. What do you think of `initiatorUrl` and `receiverUrl`?

Reply:
agreed, was trying to pick something that we are currently using e.g. requester responder, but then it was more of query part, happy to update to invitationUrl / receive